Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volumegroup-snapshot.md

```diff
@@ -0,0 +1,109 @@
+---
+title: VolumeGroup Snapshots
```

Comment:
as it is cephfs specific can we rename file with cephfs keyword

Reply:
The RBD docs will be added later in the same file. Hence the agnostic naming.

Reply:
We have plan to update this doc for rbd as well.

Reply:
`VolumeGroup Snapshots` to `Volume Group Snapshots` please change volumegroup to `volume-group` in file name as well